scripts/plot.py

```diff
@@ -0,0 +1,147 @@
+import json
+import pandas as pd
+import pathlib
+import time
+import matplotlib.pyplot as plt
+
+
+def find_latest(directory_name):
+    list_of_paths = pathlib.Path(directory_name).glob("*")
+    # list_of_dirs = list(filter(lambda p: p.is_dir(), list_of_paths))
+    list_of_dirs = []
+    for path in list_of_paths:
+        if path.is_dir():
+            list_of_dirs.append(path)
+    latest_dir = max(list_of_dirs, key=lambda p: p.stat().st_ctime)
+
+    return latest_dir, latest_dir / "wandb-events.jsonl"
+
+
+def load(directory_name):
+    directory, path = find_latest(directory_name)
+    output_dic = {}
+
+    clock = 0
+    while not path.exists():
+        clock += 1
+        time.sleep(1)
+        if clock >= 60:
+            raise FileExistsError(f"{path} is not found!")
```

Comment:
What dos this clock section do? It isn't clear how the logic in this section is going to do anything other then wait 60 seconds before raising an error.